### DISCUSSION TITLE
no app\Http\Kernel.php in the latest Laravel

### DISCUSSION THREAD

**Discussion by DaveyD1234** [spatie/laravel-multitenancy#541] [Q&A]
Hello,

New to Multi-Tenancy, when reading through the Base installation It mentions adding some code to **app\Http\Kernel.php** but looking in **app\Http** there is no Kernel.php

Where do I add the code for base Installation as per the documentation [Protecting Against Cross Tenant Abuse]
https://spatie.be/docs/laravel-multitenancy/v3/installation/base-installation#content-protecting-against-cross-tenant-abuse

If I have to create Kernel.php does anybody have a code example?

Cheers

Dave

**Comment by masterix21** [comment]
Fixed. Thanks!
